js/lib/src/state/common.ts

```diff
@@ -108,16 +108,16 @@ export class Callback {
 
 export class NamespaceAndIndex {
   namespace: web3.PublicKey;
-  indexed: boolean;
+  index: number | null;
```

Comment:
When will this be null?

Reply:
When an artifact is not indexed it will be null